CHANGELOG.md

```diff
@@ -75,11 +76,11 @@ The `paua` upgrade will increase all validators' max commission to 100% and max
   [PR 1334](https://github.com/provenance-io/provenance/pull/1334),
   [PR 1348](https://github.com/provenance-io/provenance/pull/1348),
   [PR 1317](https://github.com/provenance-io/provenance/pull/1317),
-  [PR 1278](https://github.com/provenance-io/provenance/pull/1278).  
+  [PR 1278](https://github.com/provenance-io/provenance/pull/1278). \
   See the following `RELEASE_NOTES.md` for details:
   [v0.46.10-pio-1](https://github.com/provenance-io/cosmos-sdk/blob/v0.46.10-pio-1/RELEASE_NOTES.md),
   [v0.46.8-pio-3](https://github.com/provenance-io/cosmos-sdk/blob/v0.46.8-pio-3/RELEASE_NOTES.md),
-  [v0.46.7-pio-1](https://github.com/provenance-io/cosmos-sdk/blob/v0.46.7-pio-1/RELEASE_NOTES.md).  
+  [v0.46.7-pio-1](https://github.com/provenance-io/cosmos-sdk/blob/v0.46.7-pio-1/RELEASE_NOTES.md). \
```

Comment:
In markdown, a trailing <space><space> indicates a new line. However, our `make linkify` script trims trailing spaces, causing these to be lost.
Luckily, in most markdown, you can use a `\` to indicate a new line too.

So that's why this change is here.